services/emailer.js

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv'
+import { BaseEmail } from '../data/email/base-email.js';
+// environment variables must be set before connection with db is established
+dotenv.config()
+/**
+ * Handles e-mailing from the admin account
+ * E-mail the admin when a new
+ */
+export class Emailer {
+	#sender;
+	#transporter;
+	constructor() {
+		this.#sender = process.env.ADMIN_EMAIL;
+		this.#transporter = nodemailer.createTransport({
```

Comment:
Does this create a side effect? What happens internally when createTransprot is called after new Emailer is called? Does it create a dangling connection?

Reply:
I tried to research this but didn't find anything.